release-engineering/README.md

```diff
@@ -208,6 +209,13 @@ that any final cherry-pick content is merged ahead of that last release.
   - Publishing bot updates
     - The bot runs on a schedule but sometimes breaks, requiring ad hoc fixes.
   - Curation of the [base images exception list](https://github.com/kubernetes/sig-release/blob/master/release-engineering/baseimage-exception-list.md) (most of this list is complete; more info [here](https://github.com/kubernetes/enhancements/issues/1729)).
-  - Maintaining instructions on how to consume all above official artifacts, plus variations for how 
+  - Maintaining instructions on how to consume all above official artifacts, plus variations for how
 consumers can test pre-release Alpha/Beta/RC artifacts
   - push-build.sh maintenance, until it goes away
+
+### Handbooks
+
+- [Release Managers Onboarding](./handbooks/onboarding.md)
+- [Release Management Hanbook](./handbooks/release-management.md)
+- [Branch Management Hanbook](./handbooks/branch-manager.md)
```

Comment:
```suggestion
- [Release Management Handbook](./handbooks/release-management.md)
- [Branch Management Handbook](./handbooks/branch-manager.md)
```

Reply:
Thanks for the feedback Sascha, I will continue the work and make the changes you suggested